packages/db/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.9"
+services:
+  postgres:
+    image: postgres:13
```

Comment:
I have beta access to neondatabase as a serverless postgres provider and they let me choose between v14 & 15 -- Would you like to try Neon out?
<img width="511" alt="image" src="https://user-images.githubusercontent.com/18185649/198892953-5440f01f-f3cf-4be3-81f4-200d260dd373.png">

Reply:
i don't have a preference. at first glance it looks cool 👍🏼